app/views/layouts/_user_menu.html.haml

```diff
@@ -21,7 +21,10 @@
     = link_to(destroy_user_session_path, :method=>'delete') do
       %span.fa.fa-minus
       Sign out
-- if can? :manage, Conference
+- if current_user.is_admin || (current_user.has_any_role? :admin, { :name => :organizer, :resource => :any },
```

Comment:
Yes that is what is happening in simplified words, but technically speaking? Show me the code that assigns the necessary access!